Makefile

```diff
@@ -1,34 +1,49 @@
-## Project rules
-# default rule
+########### Variables ##########
+
+EXE	=	notes-service
+
+########### !Variables #########
+
+########### Project rules ##########
+
 all:	build
-# Make if go.mod is already init
+
 build:
 	go build .
-# Create an executable for service note
-re:
+
+init:
 	rm go.mod
-	go mod init notes-service
+	go mod init ${EXE}
 	go mod tidy
```

Comment:
I don't see the goal of this rule. `go mod init` should be run once and never again?